### PR TITLE
fix Blackbox logging rate over 8kHz #415

### PR DESCRIPTION
**Pull request by EGParadox**
increase pid_rate max possible value from 8k to 32k for Blackbox logging max value calculation

### CODE REVIEW

File: src/js/tabs/onboard_logging.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ TABS.onboard_logging.initialize = function (callback) {
         
         // Offer a reasonable choice of logging rates (if people want weird steps they can use CLI)
         var loggingRates = [];
-        var pidRate = 8000 / PID_ADVANCED_CONFIG.gyro_sync_denom / PID_ADVANCED_CONFIG.pid_process_denom; 
+        var pidRate = 32000 / PID_ADVANCED_CONFIG.gyro_sync_denom / PID_ADVANCED_CONFIG.pid_process_denom;
```
Comment (mikeller) [inline src/js/tabs/onboard_logging.js]: This needs to be dependent on 32 kHz gyro updates being enabled.

**Comment by EGParadox** [comment]
i added usegyro32 flag check

File: src/js/tabs/onboard_logging.js [hunk 2]
```diff
@@ -180,7 +180,12 @@ TABS.onboard_logging.initialize = function (callback) {
         
         // Offer a reasonable choice of logging rates (if people want weird steps they can use CLI)
         var loggingRates = [];
-        var pidRate = 8000 / PID_ADVANCED_CONFIG.gyro_sync_denom / PID_ADVANCED_CONFIG.pid_process_denom; 
+        var pidRateBase = 8000;
+
+        if (PID_ADVANCED_CONFIG.gyroUse32kHz !== 0) pidRateBase = 32000;
```
Comment (mikeller) [inline src/js/tabs/onboard_logging.js]: Can you please use the form

```
if (PID_ADVANCED_CONFIG.gyroUse32kHz !== 0) {
    pidRateBase = 32000;
}
```

(All conditional statements should use blocks, and use separate lines, for maintainability's sake.)

Comment (EGParadox) [reply src/js/tabs/onboard_logging.js]: done and retested with 32k FC

**Pull request merged**